frontend/app/src/components/landingpages/blog/Governance.svelte

```diff
@@ -0,0 +1,122 @@
+<script lang="ts">
+    import ExternalLink from "../ExternalLink.svelte";
+    import Link from "../Link.svelte";
+    import BlogScreenshot from "./BlogScreenshot.svelte";
```

Comment:
Unused import I think

Reply:
Oh yes - I had intended to add screen shots